test/integration.test.ts

```diff
@@ -600,18 +601,62 @@ const testCases: {
       expect(tableHeads).toContain('Exports')
       expect(tableHeads).toContain('File')
       expect(tableHeads).toContain('Size')
+      const [exportsIndex, fileIndex, sizeIndex] = [
+        tableHeads.indexOf(pc.underline('Exports')),
+        tableHeads.indexOf(pc.underline('File')) - 9, // pc.underline('File').length = 4  + 'File'.length + 5
```

Comment:
pc.xx will modify string parameter, This is a temporary solution. Need review.

Reply:
You can use `stripANSIColor` to strip the colors, then we only test the text itself